app/javascript/src/NewService/components/FormElements/index.jsx

```diff
@@ -5,5 +5,6 @@ export {HiddenServiceDiscoveryInput} from 'NewService/components/FormElements/Hi
 export {Label} from 'NewService/components/FormElements/Label'
 export {Select} from 'NewService/components/FormElements/Select'
 export {ErrorMessage} from 'NewService/components/FormElements/ErrorMessage'
+export {SelectBackendApi} from 'NewService/components/FormElements/SelectBackendApi'
```

Comment:
I would call it `BackendApiSelect`, since it's a Select element not a method _to select_ something. Just my opinion.